rest-server/core/src/main/scala/com/actionml/core/jobs/JobManager.scala

```diff
@@ -17,17 +17,18 @@
 
 package com.actionml.core.jobs
 
-import java.util.UUID
+import java.util.{Date, UUID}
 
 import com.actionml.core.jobs.JobStatuses.JobStatus
 import com.actionml.core.model.Response
 import com.actionml.core.spark.LivyJobServerSupport
-import com.actionml.core.store.DAO
+import com.actionml.core.store.Ordering.desc
 import com.actionml.core.store.backends.MongoStorage
+import com.actionml.core.store.{DAO, OrderBy}
 import com.typesafe.scalalogging.LazyLogging
-import org.bson.{BsonInvalidOperationException, BsonReader, BsonWriter}
-import org.bson.codecs.{Codec, DecoderContext, EncoderContext}
 import org.bson.codecs.configuration.{CodecProvider, CodecRegistry}
+import org.bson.codecs.{Codec, DecoderContext, EncoderContext}
+import org.bson.{BsonInvalidOperationException, BsonReader, BsonWriter}
```

Comment:
it looks like Job Manager depends on BSON, which is Mongo only. No code should use MongoDAO or MongoStore or rely on any Mongo specific imports. These are creating technical debt. Always imagine the next feature will be Postgres instead of Mongo. In the case of MongoAdministrator, this itself is tech debt but at least the class is flagged as being Mongo specific.